backend/src/telegram/models/telegram-op.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+  BelongsTo,
+} from 'sequelize-typescript'
+import { Campaign } from '@core/models/campaign'
+import { MessageStatus } from '@core/constants'
+
+@Table({ tableName: 'telegram_ops', underscored: true, timestamps: true })
+export class TelegramOp extends Model<TelegramOp> {
+  @Column({
+    type: DataType.BIGINT,
+    autoIncrement: true,
+    primaryKey: true,
+  })
+  id!: number
+
+  @ForeignKey(() => Campaign)
+  @Column({
+    type: DataType.INTEGER,
+    allowNull: false,
+  })
+  campaignId!: number
+
+  @BelongsTo(() => Campaign)
+  campaign!: Campaign
+
+  @Column({
+    type: DataType.BIGINT,
+    allowNull: false,
+  })
+  recipient!: number
+
+  @Column(DataType.JSON)
+  params!: object
+
+  @Column(DataType.STRING)
+  messageId?: string
+
+  @Column(DataType.STRING)
+  errorCode?: string
+
+  @Column({
+    type: DataType.ENUM(...Object.values(MessageStatus)),
```

Comment:
No need to change this. Making a note here that the relevant casting was done in log-job-telegram.sql